src/imagewidget/index.css

```diff
@@ -3,19 +3,7 @@
 | Distributed under the terms of the Modified BSD License.
 |----------------------------------------------------------------------------*/
 .jp-ImageWidget {
-  overflow: scroll;
-  /*padding: 5px;*/
-}
-
-/*.jp-ImageWidget > div {
-  transform: scale(.5);
-}*/
-
-.jp-ImageWidget img {
-  max-width: 100%;
-  max-height: 100%;
-  transform: scale(.5);
-  /*display: block;*/
-  /*margin: auto;*/
-  /*object-fit: contain;*/
+  padding: 5px;
+  object-fit: contain;
```

Comment:
Unfortunately, `object-fit`, despite being the nicest solution to this issue, isn't supported by IE, so it can't be our answer for this.

[<img width="584" alt="object-fit on MDN" src="https://cloud.githubusercontent.com/assets/159529/16644666/952750d8-4416-11e6-9771-5d4f1e64b27d.png">](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit)

https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit